### DISCUSSION TITLE
Translation doubt

### DISCUSSION THREAD

**Discussion by hugok79** [SubtitleEdit/subtitleedit#5632] [General]
What is the meaning of "HI" in the string?
https://github.com/SubtitleEdit/subtitleedit/blob/6b7d70b80a4d57445a11a9722a64558ab9bb331e/LanguageMaster.xml#L2338

**Comment by niksedk** [comment]
HI = Hearing Impaired (people with any degree of hearing loss)

**Reply by hugok79** [reply]
Thank you!
Just like in the line below...
https://github.com/SubtitleEdit/subtitleedit/blob/6b7d70b80a4d57445a11a9722a64558ab9bb331e/LanguageMaster.xml#L261
